androidApp/src/main/java/com/perrigogames/life4/android/feature/settings/SettingsScreen.kt

```diff
@@ -41,6 +43,26 @@ fun SettingsScreen(
             onAction = { viewModel.handleAction(it) }
         )
     }
+    LaunchedEffect(Unit) {
+        viewModel.events.collect { event ->
+            when(event) {
+                is SettingsEvent.WebLink -> {
+                    Log.v("Settings", "Opening web link ${event.url}")
+                    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(event.url))
+                    context.startActivity(intent)
+                }
+                is SettingsEvent.Email -> {
+                    Log.v("Settings", "Opening email client to ${event.email}")
+                    val emailUri = Uri.parse(
+//                        "mailto:${event.email}?subject=${Uri.encode(subject)}&body=${Uri.encode(body)}"
```

Comment:
```suggestion
                        "mailto:${event.email}?subject=${Uri.encode(subject)}&body=${Uri.encode(body)}"
```

Reply:
This is commented because currently there is not a decided template to follow here, but I want to make it clear that these items are available at this spot in the code.